internals/schema_migration.py

```diff
@@ -18,6 +18,33 @@
 from framework.basehandlers import FlaskHandler
 from internals.review_models import Activity, Comment
 
+def handle_migration(original_cls, new_cls, migration_cls, kwarg_mapping):
+  originals = original_cls.query().fetch()
+  new_keys = new_cls.query().fetch(keys_only=True)
+  new_ids = set(key.integer_id() for key in new_keys)
+  migration_count = 0
+  for original in originals:
+    # Check if a new entity with the same ID has already been created.
+    # If so, do not create the entity again.
+    if original.key.integer_id() in new_ids:
+      continue
+
+    kwargs = {k : getattr(original, v) for (k, v) in kwarg_mapping}
```

Comment:
The variables k and v could be more clear.  E.g., `(new_field, old_field)`.

Reply:
True, fixed.